clicmd

```diff
@@ -31,10 +31,14 @@ if [[ "${USER}" != "root" || ! -v GROUP ]]; then
   fi
   GROUP=$(id -Gn | sed -n 's/.*\(%GROUP_REGEX%\)[ ]*.*$/\1/gp')
 fi
+if [[ -z "${GROUP}" ]]; then
+  echo "Can't find group for user ${USER}" >&2
+  exit 2
+fi
 
 # Lookup for group by role name (privilege in BMC terms)
 function role_to_group {
-  case "$1" in
+  case "${1:-}" in
```

Comment:
The same case as in the `group_to_role`.
The error message at line 46  should be fixed.